src/epoll.rs

```diff
@@ -65,11 +65,11 @@ impl Poller {
             PollMode::Oneshot,
         )?;
 
-        log::trace!(
-            "new: epoll_fd={}, event_fd={}, timer_fd={:?}",
-            poller.epoll_fd.as_raw_fd(),
-            poller.event_fd.as_raw_fd(),
-            poller.timer_fd
+        tracing::trace!(
+            epoll_fd= ?poller.epoll_fd.as_raw_fd(),
+            event_fd= ?poller.event_fd.as_raw_fd(),
+            timer_fd= ?poller.timer_fd,
```

Comment:
```suggestion
            epoll_fd = ?poller.epoll_fd.as_raw_fd(),
            event_fd = ?poller.event_fd.as_raw_fd(),
            timer_fd = ?poller.timer_fd,
```

(Same as https://github.com/smol-rs/async-io/pull/140#discussion_r1228936839)